deployment/keystone/changeset/view.go

```diff
@@ -14,7 +14,12 @@ import (
 
 var _ deployment.ViewState = ViewKeystone
 
+var _ deployment.StateRenderer[view.KeystoneView] = RenderKeystoneState
+
 func ViewKeystone(e deployment.Environment) (json.Marshaler, error) {
+	return RenderKeystoneState(e)
+}
+func RenderKeystoneState(e deployment.Environment) (*view.KeystoneView, error) {
```

Comment:
We could consider using `Operations` for constructing views. An operation can also be used to query, it would unify devs APIs, would have a way to be composed, and could have similar APIs to be consumed